org.jacoco.doc/docroot/doc/changes.html

```diff
@@ -76,8 +76,8 @@ <h3>Non-functional Changes</h3>
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/1468">#1468</a>).</li>
   <li>JaCoCo build now requires JDK 17
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/1482">#1482</a>).</li>
-  <li>JaCoCo now depends on ASM 9.6
-      (GitHub <a href="https://github.com/jacoco/jacoco/issues/1518">#1518</a>).</li>
+  <li>JaCoCo now depends on ASM 9.7
```

Comment:
We can't modify the change log of released JaCoCo version 0.8.11. Please add on top under "Snapshot...". Thanks!

Reply:
I should have looked at the context to see that one needs to add new entries, not modify historical entries.  😱